data_processing/getPathologyAnnotations/getPathologyAnnotations.py

```diff
@@ -47,14 +53,18 @@ def getSlideIDs_case(input_id):
 """
 curl http://<server>:5001/mind/api/v1/datasets/MY_DATASET
 """
-@app.route('/mind/api/v1/getPathologyAnnotation/<string:project>/<string:slide_hid>/<labelset>', methods=['GET'])
-def getPathologyAnnotation(project,slide_hid,labelset):
+@app.route('/mind/api/v1/getPathologyAnnotation/<string:project>/<string:slide_hid>/<string:annotation_type>/<labelset>', methods=['GET'])
+def getPathologyAnnotation(annotation_type, project,slide_hid,labelset):
 	slide_id = get_slide_id(slide_hid, "slide_hid")
 
-	spark = SparkConfig().spark_session(os.environ['SPARK_CONFIG'], "data_processing.getPathologyAnnotation")
-
 	ANNOTATIONS_FOLDER = os.path.join("/gpfs/mskmindhdp_emc/data/pathology", project, "annotations")
```

Comment:
we should not have paths hardcoded in this mannner. All of this information should come from app config and the key value pairs in that config template should be well documented to state what they are supposed to be. For example, in this case the config could be setup as follows:

```
pathology:

  # root path to pathology data in the data lake
  - root_path: /gpfs/mskmindhdp_emc/data/pathology
```